sql_script.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE inventory (
     itemID INT PRIMARY KEY,
     userID INT NOT NULL,
     analyticsID INT NOT NULL,
-    itemName VARCHAR(20) NOT NULL

```

Comment:
Use the current sql_script. A new one will be uploaded soon, whose only change will be in orders

Reply:
Reverted.